template/page-events.html.jinja

```diff
@@ -328,6 +328,26 @@
     "image": "AMAs.png",
     "end_date": "March 15, 2023"
     },
+    {
+    "name": "XRPL Community Meetup: Madrid",      
+    "description": "We're excited to host our second Miami meetup for XRP Ledger community members on April 6th from 6-8pm, featuring Marco Neri, Developer Advocate at Ripple, who will join us to give a presentation on the XRP Ledger.",
+    "type": "meetup",
+    "link": "https://www.meetup.com/xrp-ledger-espana-madrid-y-barcelona/events/292597878",
+    "location": "Madrid",
+    "date": "April 29, 2023",
+    "image": "Madrid.png",
+    "end_date": "April 29, 2023" 
+    },
+    {
+    "name": "APEX 2023: The XRPL Developer Summit",      
+    "description": "Apex XRPL Developer Summit is the annual event where developers, contributors, and thought leaders come together to learn, build, share, network, and celebrate all things XRP Ledger.",
+    "type": "ama",
```

Comment:
should be type: conference, shouldn't it?

Reply:
So, they added new event types and filters. See attached 
![image](https://user-images.githubusercontent.com/43763415/236895714-a285a6de-4776-45ce-a010-8e27180c99e0.png)

Reply:
Nvm, I overlooked this, you are right it is indeed a conference, nice catch